projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/LLVMContext.java

```diff
@@ -49,6 +49,7 @@
 
     private final List<RootCallTarget> staticInitializers = new ArrayList<>();
     private final List<RootCallTarget> staticDestructors = new ArrayList<>();
+    private final List<RootCallTarget> constructorFunctions = new ArrayList<>();
```

Comment:
I think the `constructorFunctions` here contains the same variables as the `staticInitializers` in `LLVMContext`, `LLVMGlobalRootNode` and `LLVMVisitor` (see [`LLVMVisitor in line 415`](https://github.com/graalvm/sulong/blob/6c78c9b8afad7a5fb1440959c43812af4ff40ab3/projects/com.oracle.truffle.llvm.parser.impl/src/com/oracle/truffle/llvm/parser/impl/LLVMVisitor.java#L415)). The name `staticInitializers` is misleading and should probably be called `staticConstructors` so that it is obvious that they relate to the `staticDestructors`. I think for this commit it is sufficient to keep the changes in the binary bitcode parser classes.